src/v0/destinations/sfmc/transform.test.js

```diff
@@ -0,0 +1,179 @@
+const { ConfigurationError } = require('@rudderstack/integrations-lib');
+const axios = require('axios');
+const MockAxiosAdapter = require('axios-mock-adapter');
+const { responseBuilderSimple } = require('./transform');
+beforeAll(() => {
+  const mock = new MockAxiosAdapter(axios);
+  mock
+    .onPost('https://yourSubDomain.auth.marketingcloudapis.com/v2/token')
+    .reply(200, '{"access_token":"yourAuthToken"}');
+});
+
+describe('responseBuilderSimple', () => {
+  it('should return an array of two payloads for identify calls when createOrUpdateContacts is false', async () => {
```

Comment:
Do you think we need `identify` test-cases as part of this PR ?

Reply:
Not really but it doesn't harm to add :P